From: Samuel Henrique <samueloph@debian.org>
Date: Fri, 12 Apr 2024 00:11:07 +0100
Subject: Fix manpage installation for rrsync

Otherwise we would get "ERROR: support/rrsync.1 cannot be created."
I'm not confident this is the best approach on solving this issue,
but I know this works with no regressions.
This patch needs to be reviewed before being submitted to upstream.
===================================================================
---
 maybe-make-man | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/maybe-make-man b/maybe-make-man
index c7af739..d0b9660 100755
--- a/maybe-make-man
+++ b/maybe-make-man
@@ -23,12 +23,8 @@ if [ ! -f "$flagfile" ]; then
 	    cp -p "$srcdir/$outname" .
 	    exit 0
 	else
-	    echo "ERROR: $outname cannot be created."
-	    if [ -f "$HOME/build_farm/build_test.fns" ]; then
-		exit 0 # No exit errorno to avoid a build failure in the samba build farm
-	    else
-		exit 1
-	    fi
+		"$srcdir/md-convert" "$srcdir/$inname"
+	    cp -p "$srcdir/$outname" .
 	fi
     fi
 fi
